integration/spark/app/integrations/sparkrdd/1.json

```diff
@@ -16,12 +16,13 @@
     }
   },
   "job" : {
-    "namespace" : "ns_name",
-    "name" : "test_rdd.shuffled_map_partitions_hadoop"
```

Comment:
is this because the name will change every time? should that check a pattern of some sort?

Reply:
the name just changed, it wasn't necessary to remove it.